docs/content/querying/scan-query.md

```diff
@@ -198,8 +200,27 @@ is complete.
 
 ## Configuration Properties
 
+Configuration properties:
+
 |property|description|values|default|
 |--------|-----------|------|-------|
 |druid.query.scan.maxRowsQueuedForOrdering|The maximum number of rows returned when time ordering is used|An integer in [0, 2147483647]|100000|
 |druid.query.scan.maxSegmentPartitionsOrderedInMemory|The maximum number of segments scanned per historical when time ordering is used|An integer in [0, 2147483647]|50|
-|druid.query.scan.legacy|Whether legacy mode should be turned on for Scan queries|true or false|false|
\ No newline at end of file
+|druid.query.scan.legacy|Whether legacy mode should be turned on for Scan queries|true or false|false|
+
+
+## Query Context Properties
+
+|property|description|values|default|
+|--------|-----------|------|-------|
+|maxRowsQueuedForOrdering|The maximum number of rows returned when time ordering is used.  Overrides the identically named config.|An integer in [0, 2147483647]|100000|
```

Comment:
The default here would be the configured value of `druid.query.scan.maxSegmentPartitionsOrderedInMemory` instead of 100000